viewer/src/utilities/networking/ModelDataRetriever.ts

```diff
@@ -2,7 +2,39 @@
  * Copyright 2020 Cognite AS
  */
 
+import { HttpError } from '@cognite/sdk';
+
 export interface ModelDataRetriever {
   fetchJson(filename: string): Promise<any>;
   fetchData(filename: string): Promise<ArrayBuffer>;
 }
+
+export class ByUrlModelDataRetriever implements ModelDataRetriever {
```

Comment:
What is the point of this class? I didn't see it used anywhere. And why is this better than the LocalUrlClient & Cognite3dExtension?
This one seems to go back to the same design where we have one retriever per every model that gets added each with its own baseUrl.

Reply:
I didn't add it - it was in the code base you checked in. I just moved the class. If we can remove it, go ahead, @j-bjorne.

Reply:
Awesome xD it must have been a file that wasn't deleted since it was moved in the pr. I had originally deleted it, but the merge must have kept it